### ISSUE TITLE
UI Issue Latest iOS Version 

### ISSUE THREAD

**Issue by serkaneken** [kishikawakatsumi/PEPhotoCropEditor#102]
Hello, when we were upgrading our devices' ios version to the latest release happening the crops header top broke down,  is there any solution,  ı added photos as an attachment,

thanks in advance :) Happy coding :)

![0f52f8cb-5bd3-4ea4-8852-177703adf014](https://user-images.githubusercontent.com/30027922/187933885-ce96001d-abe3-40a9-8593-d42b293f5437.jpg)
![Image20220829131152](https://user-images.githubusercontent.com/30027922/187933892-235b01bd-a1a6-4c1f-86c7-72efde2f9256.jpg)
![Image20220829131221](https://user-images.githubusercontent.com/30027922/187933902-2712231e-ad7e-4709-8e28-f2dbdef67957.jpg)

**Comment by serkaneken** [comment]
We talked with @kishikawakatsumi and he said: I looked at my code, and my library provides just a crop UI and does nothing for the navigation bar area. So the problem cannot be fixed on my library side. I think you need to fix or request to fix the plugin side."

**Comment by serkaneken** [comment]
We talked with @kishikawakatsumi and he said: I looked at my code, and my library provides just a crop UI and does nothing for the navigation bar area. So the problem cannot be fixed on my library side. I think you need to fix or request to fix the plugin side."

**Comment by serkaneken** [comment]
We talked with @kishikawakatsumi and he said: I looked at my code, and my library provides just a crop UI and does nothing for the navigation bar area. So the problem cannot be fixed on my library side. I think you need to fix or request to fix the plugin side."

**Comment by iPMisterX** [comment]
### **PECropViewController.m**

Add this code
```
if (@available(iOS 13.0, *)) {
    UINavigationBarAppearance *appearance = [[UINavigationBarAppearance alloc] init];
    appearance.titleTextAttributes = @{NSForegroundColorAttributeName: [UIColor whiteColor]};
    self.navigationController.navigationBar.standardAppearance = appearance;
    self.navigationController.navigationBar.scrollEdgeAppearance = appearance;
}
```
![Screen Shot 2565-10-10 at 17 04 02](https://user-images.githubusercontent.com/11723453/194842188-85a2c380-c0f3-43a3-8f5b-22714a8e22ef.png)
